network/p2p/client.go

```diff
@@ -14,9 +14,8 @@ import (
 )
 
 var (
-	ErrAppRequestFailed = errors.New("app request failed")
```

Comment:
This used to be an error we would propagate for a timeout. Since we now have `common.ErrTimeout`, this is no longer needed.